--- a/opt/victronenergy/gui/qml/PageSettings.qml
+++ b/opt/victronenergy/gui/qml/PageSettings.qml
@@ -140,6 +140,12 @@
 			show: ioSettings.haveSubMenus
 			PageSettingsIo { id: ioSettings }
 		}
+		// ExpanderPiSetup: Hardware submenu for the DBus ADC page
+		MbSubMenu {
+			description: qsTr("ExpanderPi")
+			subpage: Component { PageSettingsExpanderPiDbusAdc {} }
+			showAccessLevel: User.AccessInstaller
+		}
 
 		/*
 		MbSubMenu {
